Entities/Industry/Drill/Drill.as

```diff
@@ -191,7 +197,7 @@ void onTick(CBlob@ this)
 		AimAtMouse(this, holder); // aim at our mouse pos
 
 		// cool faster if holder is moving
-		if (heat > 0 && holder.getShape().vellen > 0.01f && getGameTime() % 3 == 0)
+		if (heat > 0 && holder.getShape().vellen > 0.01f && getGameTime() % 4 == 0)
```

Comment:
Should we even keep this mechanic at all since it is rather obscure imo?

Reply:
why not? it doesn't really change that much but it's pretty cool to have as a little detail imo

Reply:
There's no immediate feedback that it is possible, so this is one of the (various) things drills have that you need to be told about to know.

Reply:
removed